mods/persistence/controllers/subsystems/initialization/chargen.dm

```diff
@@ -28,16 +27,99 @@ SUBSYSTEM_DEF(chargen)
 				CHECK_TICK
 
 /datum/controller/subsystem/chargen/proc/get_spawn_turf()
```

Comment:
Is this proc even referenced anywhere anymore?

Reply:
Removed it now